app/controllers/application_controller.rb

```diff
@@ -1,5 +1,12 @@
 class ApplicationController < ActionController::Base
-  # Prevent CSRF attacks by raising an exception.
-  # For APIs, you may want to use :null_session instead.
+  before_action :configure_permitted_parameters, if: :devise_controller?
   protect_from_forgery with: :exception
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:sign_up, keys: [:name, :email, :password])
+    devise_parameter_sanitizer.permit(:sign_in, keys: [:name, :email, :password])
+    devise_parameter_sanitizer.permit(:account_update, keys: [:name, :email, :password])
+  end
```

Comment:
One of these lines is repeated. Please remove!

Reply:
Aren't sign_up, sign_in, and account_update all needed?